modules/search_api_solr_autocomplete/tests/src/Kernel/SearchApiSolrOldTest.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace Drupal\Tests\search_api_solr_autocomplete\Kernel;
+
+use Drupal\search_api\Entity\Server;
+use Drupal\search_api_autocomplete\Entity\Search;
+use Drupal\Tests\search_api_solr\Kernel\SearchApiSolrTest;
+
+/**
+ * Tests search autocomplete support and ngram results using the Solr search backend.
+ *
+ * @group search_api_solr
+ */
+class SearchApiSolrOldTest extends SearchApiSolrTest {
```

Comment:
`SearchApiSolrOldTest`? I don't get this naming. Maybe `SearchApiSolrAutocompleteAndNgramTest`?